api/src/Command/UserRetentionPolicyCommand.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Command;
+
+use App\Entity\User;
+use App\Event\UserRetentionPolicyCommandEvent;
+use App\EventDispatcher\ObservableEventDispatcher;
+use App\Repository\UserRepository;
+use App\Service\Audit\AuditEvents;
+use App\v2\Controller\ControllerTrait;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class UserRetentionPolicyCommand extends Command
+{
+    use ControllerTrait;
+
+    public static $defaultName = 'digideps:user-retention-policy';
+
+    public function __construct(
+        private UserRepository $userRepository,
+        private ObservableEventDispatcher $eventDispatcher,
+        private LoggerInterface $logger
+    ) {
+        parent::__construct();
+    }
+
+    protected function configure()
+    {
+        $this
+            ->setDescription('Deletes inactive admin user accounts')
+        ;
+    }
+
+      protected function execute(InputInterface $input, OutputInterface $output)
+      {
+          /** @var User $user */
+          $getInactiveAdminUsers = $this->userRepository->getAllAdminAccountsNotUsedWithin('-24 months');
+
+          $inactiveAdminUserIds = [];
+
+          if ($getInactiveAdminUsers) {
```

Comment:
Redundant `if` since you are checking if `$inactiveAdminUserIds` is not empty further down & the additional type checking happening before populating this array, makes it not needed.